packages/ui-core/src/components/Header/Header.tsx

```diff
@@ -36,19 +45,22 @@ class Header extends React.Component<IHeaderProps, {}> {
     };
 
     renderAddition() {
-        return (
-            <div className={cn('addition')}>
-                {this.props.addition}
-            </div>
-        );
+        return <div className={cn('addition')}>{this.props.addition}</div>;
     }
 
     render() {
         const ElementType = this.props.as as string;
-        const { color, margin, as: level, onClick } = this.props;
+        const { color, margin, as: level, onClick, dataAttrs } = this.props;
 
         return (
-            <ElementType className={cn('', { color, margin, level }, this.props.className)} onClick={onClick}>
+            <ElementType
+                {...filterDataAttrs(dataAttrs)}
+                className={cn(
+                    { color, margin, level },
+                    this.props.className
```

Comment:
```suggestion
                    className
```
Выше же есть деструктуризация пропсов.